src/DSP.jl

```diff
@@ -2,14 +2,37 @@ __precompile__()
 
 module DSP
 
+# We want to be very sure we don't pull in Base names unless we're very sure we want them
+# This macro will be called in each submodule herein to do the appropriate imports
+macro importffts()
+    quote
+        using AbstractFFTs, FFTW
+        importall AbstractFFTs, FFTW
+        if VERSION >= v"0.7.0-DEV.585"
```

Comment:
there have been commits to base since though, where this shouldn't apply yet.

could try running one of the base functions then instead of checking isdefined